interface/src/app/node/node.tpl.html

```diff
@@ -0,0 +1,35 @@
+<h1>ADAGE: Node Information</h1>
+
+<div ng-if="!ctrl.statusMessage">
```

Comment:
Yes, if **statusMessage** is not blank, the whole page (including all sub-sections) will be hidden, because every sub-section depends on a valid node. When **statusMessage** is non-blank, the retrieval of the node's basic information (node name, machine learning model) must have failed, so it doesn't make sense to show any other information that is directly related to this node.

On the other hand, each section has (or will have) its own status. For example, in **Heavy Weight Gens** section, a **queryStatus** property is defined to indicate this section's query status. If it is not blank, only the corresponding section will be hidden, and it won't affect any other sections.